component/main.jsonnet

```diff
@@ -1,16 +1,27 @@
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
+local com = import 'lib/commodore.libjsonnet';
 
 local params = inv.parameters.grafana_helm;
 
+local secrets = com.generateResources(
+  params.secrets,
+  function(name) kube.Secret(name) {
+    metadata+: {
+      namespace: params.namespace,
+    },
```

Comment:
technically not required, assuming the target namespace of the ArgoCD app is set correctly.